### ISSUE TITLE
Node Drag Handle 

### ISSUE THREAD

**Issue by datoslabs** [wbkd/react-flow#1413]
Hi,

Would it be possible to add `handle` selector property to Node options so that it can be passed to DraggableCore component in [wrapNode.tsx](https://github.com/wbkd/react-flow/blob/64d1913356d171fd2c4303c918c3e375f57cbee7/src/components/Nodes/wrapNode.tsx#L234-L244)?  I would like to create a custom node where user can only drag the node via the drag handle, please let me know if there is an alternative way to achieve this.  Thank you!

**Comment by datoslabs** [comment]
I have an advanced use case to create a custom node that can encapsulate other nodes (like a node group or in-line nested sub-flow); in order for users to freely drag external nodes into and out of this custom node as well moving the nodes together as a group, it requires a designated handle (`dragHandle`) to initiate drag thus freeing up the remaining space outside of `dragHandle` free to respond to other pointer events.  Currently, every node is wrapped inside a `DraggableCore` component thus marking the entire node area responsive to drag event.  DragHandle is a common requirement/functionality provided by most DnD libraries including react-draggable.

This enhancement allows users to specify optional `dragHandle` selector property which is provided to `DraggableCore` component's `handle` property to designate a rendering component responsive to the initiation of drag event.

Usage:
```
[
  {
    id: '1',
    type: 'input',
    data: {
      label: (
        <>
          Welcome to <strong>React Flow!</strong>
        </>
      ),
    },
    position: { x: 250, y: 0 },
  },
  {
    id: '2',
    data: {
      label: (
        <>
          This is a <strong>default node</strong>
        </>
      ),
    },
    position: { x: 100, y: 100 },
    dragHandle: 'strong', //node drag will only initiate when cursor is over <strong>default node</strong>
  },
...
]
```
I have submitted a PR with this enhancement.

**Comment by datoslabs** [comment]
Released in V9.6.8.
